src/v/config/node_config.cc

```diff
@@ -120,14 +120,50 @@ node_config::node_config() noexcept
       {.visibility = visibility::user},
       {}) {}
 
+void validate_multi_node_propert_config(
```

Comment:
Looks fine, but needs a test; or maybe my eyes are tired.

Reply:
As mentioned on the cover letter - tests are incoming, but I'm too stupid to be able to write them.

Reply:
See #5906